packages/plugin-errors/README.md

```diff
@@ -311,7 +311,6 @@ BEFORE the validation plugin in your plugin list.
 If a field with `errors` returns a `loadableObject`, or `loadableNode` the errors plugin will now
 catch errors thrown when loading ids returned by the `resolve` function.
 
-plugin. This is because if items are nullable, the items in the list may be set to null rather that
```

Comment:
I noticed this line didn't make sense, but I wasn't sure if it was still needed so I deleted it. If it is still relevant, let me know what it should say and I can update it.

Reply:
Yeah, not entirely sure where that came from... I think good to remove for now.  thanks!